types/tailwindcss/defaultTheme.d.ts

```diff
@@ -0,0 +1,5 @@
+import type { TailwindConfig } from './tailwind-config';
+
+declare const theme: TailwindConfig;
+
+export default theme;
```

Comment:
```suggestion
export = theme;
```
https://github.com/tailwindlabs/tailwindcss/blob/master/defaultTheme.js#L4

Reply:
done so thanks